lib/queries/useAuthoritiesBrowse/useAuthoritiesBrowse.js

```diff
@@ -162,7 +164,11 @@ const useAuthoritiesBrowse = ({
     resetPageCache,
   } = useBrowserPaging(searchQuery);
 
+  const qindex = queryString.parse(location.qindex);
```

Comment:
Getting value from `location` isn't going to work in the plugin, because in that case we don't change the url
Could you try to pass `searchIndex` to `useBrowserPaging`, and then on line 138 add `searchIndex` to dependencies array? It should trigger the reset of page